packages/klighd-core/src/sidebar/sidebar-panel.ts

```diff
@@ -15,9 +15,21 @@
  * SPDX-License-Identifier: EPL-2.0
  */
 
-import { injectable } from "inversify";
+import { inject, injectable } from "inversify";
 import { VNode } from "snabbdom";
 
+//rik imports
+import { RefreshDiagramAction } from "@kieler/klighd-interactive/lib/actions";
+import { CenterAction } from "sprotty-protocol";
+import { KlighdFitToScreenAction, RefreshLayoutAction } from "../actions/actions";
+import { CreateBookmarkAction } from "../bookmarks/bookmark";
+import { PossibleQuickAction, QuickActionOption } from "../options/option-models";
+import { SetRenderOptionAction } from "../options/actions";
+import { PinSidebarOption,RenderOptionsRegistry, ResizeToFit } from "../options/render-options-registry";
+import { IActionDispatcher, RequestExportSvgAction, TYPES } from "sprotty";
+import { DISymbol } from "../di.symbols";
```

Comment:
no need to mention that these imports were added by you.
Please also sort the import lines alphabetically, i.e. sort "sprotty" above to "sprotty-protocol". See other files for reference.